### DISCUSSION TITLE
Browser In The Browser (BITB) Templates: Possible templates for HE:RE

### DISCUSSION THREAD

**Discussion by Morsmalleo** [D4rkS3c-Group/HiddenEyeReborn#76] [Ideas] [answered]
G'day guys, I found this repository while doing some browsing, well actually it came up on my news feed on my phone, hope it helps you guys out with HE:RE.

It's not my work either for the record, I just seen it and thought it might help you guys out, it's used for Phishing Attacks and is apparently pretty good to.

https://github.com/mrd0x/BITB

**Comment by sTiKyt** [comment, marked as answer]
Very interesting, i'll look into it after war is over
